src/Scrabble/Move/Move.hs

```diff
@@ -42,9 +42,13 @@ createMove' :: Validator -- ^
 createMove' validate b (Rack rack) wp dict = if valid then go else errMsg where
   errMsg        = Left "error: rack missing input letters"
   rackLetters   = fmap letter rack
-  valid         = containsAll (toString putLetters) (toString rackLetters)
+  valid         = containsAllWithBlanks (toString putLetters) (toString rackLetters)
+  blanksUsed    = countBlanksUsed (toString putLetters) (toString rackLetters)
```

Comment:
Unless there is a compiler optimization that I'm unaware of that detects equal expressions, this won't magically work. They are two independent expressions/computations, not a pointer/name to a single computation.